Exchange/ExchangeView/ExchangeViewController.swift

```diff
@@ -291,42 +253,42 @@ final class ExchangeViewController: UIViewController {
 extension ExchangeViewController: ViewType {
     typealias ViewModel = ExchangeViewModel
     
+    var bindings: ViewModel.Bindings {
+        .init(
+            didTapFirstCurrencySelectionButton: firstCurrencySelectionButton.rx.tap.asSignal(),
+            didTapSecondCurrencySelectionButton: secondCurrencySelectionButton.rx.tap.asSignal(),
+            textOfFirstCurrencyTextField: firstCurrencyTextField.rx.text.asDriver(),
+            textOfSecondCurrencyTextField: secondCurrencyTextField.rx.text.asDriver()
+        )
+    }
+    
     func bind(to viewModel: ExchangeViewModel) {
+        viewModel.firstCurrency
+            .drive(onNext: { [weak self] currency in
+                self?.firstCurrencyLabel.text = currency
+            })
```

Comment:
Привыкай к короткой записи. Тут и ниже
```suggestion
            .drive(firstCurrencyLabel.rx.text)
```

Reply:
исправил